app/views/devise/registrations/new.html.haml

```diff
@@ -0,0 +1,14 @@
+%h2 Create an account
+%h5 It's free and always will be.
+%br
+= simple_form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f|
+  .form-inputs
+    = f.input :given_name, label: "First Name", required: true, autofocus: true
+    = f.input :last_name, label: "Last Name", required: true
+    = f.input :brith_date, label: "Date of Birth", required: true
+    = f.input :email, label: "Email", required: true
+    = f.input :password, label: "Password", required: true, hint: ("#{@minimum_password_length} characters minimum" if @validatable)
+    = f.input :password_confirmation, label: "Confirm Password", required: true
+  .form-actions
+    = f.button :submit, "Sign up"
+    = render "devise/shared/links"
```

Comment:
@lemattma I've just updated all the device related erb's to haml files. Please let me know if it needs any corrections ?